weblate/trans/actions.py

```diff
@@ -0,0 +1,166 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from django.db.models import IntegerChoices
+from django.utils.translation import gettext_lazy, pgettext_lazy
+
+
+class ActionEvents(IntegerChoices):
+    # Translators: Name of event in the history
```

Comment:
This needs to be before every line so that it is included in every translation string.